.rubocop.yml

```diff
@@ -4,9 +4,17 @@ AllCops:
 
 require:
   - rubocop-rspec
+  - rubocop
+  - rubocop-performance
 
 RSpec/HookArgument:
   EnforcedStyle: each
 
+RSpec/InstanceVariable:
+ Enabled: false
+
 Metrics/LineLength:
-  Max: 120
\ No newline at end of file
+  Max: 200
+
+Metrics/ClassLength:
+  Max: 200
```

Comment:
I need more length 220 will be good

Reply:
The requested change will be resolved on the next pull request.